src/ui/FacetValueSuggestions/FacetValueSuggestionsProvider.ts

```diff
@@ -97,7 +98,7 @@ export class FacetValueSuggestionsProvider implements IFacetValueSuggestionsProv
     const referenceValuesRequest = this.buildReferenceFieldValueRequest();
     const queryParts = this.getQueryToExecuteParts();
     const suggestionValuesRequests = valuesToSearch.map(keyword =>
-      this.buildListFieldValueRequest([...queryParts, keyword.text].join(' '))
+      this.buildListFieldValueRequest([...queryParts, ...[this.options.expression].filter(exp => !!exp), keyword.text].join(' '))
```

Comment:
Like Sami pointed out, using more lines and variables would be more readable here.

Reply:
Yeap. A good ol' `if` and an extra variable will do it.